specs/order_spec.rb

```diff
@@ -141,21 +176,52 @@
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      id = 100
+      products = {
+        "Amaranth" => 83.81,
+        "Smoked Trout" => 70.6,
+        "Cheddar" => 5.63,
+      }
+      customer_id = 20
+      # fulfillment_status = :pending
+
+      order = Order.all.last
+
+      expect(order.id).must_equal id
+      expect(order.products).must_equal products
+      expect(order.customer).must_be_kind_of Customer
+      expect(order.customer.id).must_equal customer_id
+      expect(order.fulfillment_status).must_equal :pending
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+      first = Order.find(1)
+
+      expect(first).must_be_kind_of Order
+      expect(first.id).must_equal 1
     end
 
     it "Can find the last order from the CSV" do
-      # TODO: Your test code here!
+      last = Order.find(100)
+
+      expect(last).must_be_kind_of Order
+      expect(last.id).must_equal 100
     end
 
     it "Returns nil for an order that doesn't exist" do
-      # TODO: Your test code here!
+      expect(Order.find(53145)).must_be_nil
+    end
+  end
+
+  describe "Order.find_by_customer" do
+    it "Can return list of orders" do
+      found = Order.find_by_customer(20)
+
+      expect(found.last).must_be_kind_of Order
+      expect(found.first).must_be_kind_of Order
+      expect(found).must_be_kind_of Array
     end
```

Comment:
Nice work on the optional `find_by_customer`! I would probably expect a test to check what happens if the given ID isn't an existing customer ID